repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/intrafilterlogging/header-normalization.cfg.xml

```diff
@@ -22,14 +22,6 @@
             </whitelist>
         </target>
 
-        <!-- applies to all POST and PUT requests -->
-        <target http-methods="POST PUT">
-            <whitelist id="modification">
-                <header id="X-THIRD-Filter"/>
-                <header id="x-auth-token"/>
-            </whitelist>
-        </target>
-
```

Comment:
Interesting. My worry would just be that the way two filters work together prevent us from catching an issue with the component under test, or that we end up testing everything with multiple filters and when one has a bug we have to fix it across a huge number of tests.

I think ideally, we would test every component in isolation and in a production-type environment. Both approaches have their pros and cons. In the end, this works, so I have no real complaints.